src/common/pf_lldp.c

```diff
@@ -440,10 +533,239 @@ void pf_lldp_send(
 void pf_lldp_init(
    pnet_t                  *net)
 {
+   memset(&net->lldp_peer_info, 0, sizeof(net->lldp_peer_info));
+
    pf_lldp_send(net);
    if (pf_scheduler_add(net, PF_LLDP_INTERVAL*1000,
-      lldp_sync_name, pf_lldp_trigger_sending, NULL, &net->lldp_timeout) != 0)
+      shed_tag_tx, pf_lldp_trigger_sending, NULL, &net->lldp_timeout) != 0)
    {
       LOG_ERROR(PF_ETH_LOG, "LLDP(%d): Failed to reschedule LLDP sending\n", __LINE__);
    }
 }
+
+/**
+ * @internal
+ * Parse LLDP frame buffer
+ *
+ * @param buf              :In Reference to LLDP data
```

Comment:
The stack uses this convention:
@param port_id          In:   Peer port ID. Terminated string.

Reply:
OK